frontend/src/main/scala/bloop/engine/tasks/RunTasks.scala

```diff
@@ -0,0 +1,38 @@
+package bloop.engine.tasks
+
+import bloop.Project
+import bloop.cli.ExitStatus
+import bloop.engine.State
+import bloop.exec.ProcessConfig
+
+import sbt.internal.inc.Analysis
+import xsbt.api.Discovery
+
+object RunTasks {
```

Comment:
Do you think it would be better to move this to `CompileTasks` and rename that file to `Tasks`? That file is already misnamed, so I think this is the way to go -- centralizing all the implementations of these core tasks is better IMO.